config-handler-router/pom.xml

```diff
@@ -44,6 +44,60 @@
             <artifactId>commons-lang</artifactId>
             <version>${commons-lang.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>${guava.version}</version>
+        </dependency>
+
+        <!--
+    TODO: Replace all dependencies below this line with shaded jar at version 2.10.0
```

Comment:
Can we move all these groovy dependencies to the top so if new ones are added, they don't automatically get grouped together by accident. I also like to put a comment at the end of a specific set of dependencies like <!-- end Spock dependencies-->